qutip/solver/brmesolve.py

```diff
@@ -79,19 +81,26 @@ def brmesolve(
                 (c+c.dag(), SpectraCoefficient(coefficient(array, tlist=ws))),
             ]
 
-        .. note:
-            ``Cubic_Spline`` has been replaced by :obj:`.Coefficient`:
+        .. note::
+
+            ``Cubic_Spline`` has been replaced by:
                 ``spline = qutip.coefficient(array, tlist=times)``
 
-            Whether the ``a_ops`` is time dependent is decided by the type of
-            the operator: :obj:`.Qobj` vs :obj:`.QobjEvo` instead of the type
-            of the spectra.
+        Whether the ``a_ops`` is time dependent is decided by the type of
+        the operator: :obj:`.Qobj` vs :obj:`.QobjEvo` instead of the type
+        of the spectra.
 
-    e_ops : list, dict, :obj:`.Qobj` or callback function, optional
-        Single operator, or list or dict of operators, for which to evaluate
-        expectation values. Operator can be Qobj, QobjEvo or callables with the
-        signature `f(t: float, state: Qobj) -> Any`.
-        Callable signature must be, `f(t: float, state: Qobj)`.
+    sec_cutoff : float, default: 0.1
+        Cutoff for secular approximation. Use ``-1`` if secular approximation
+        is not used when evaluating bath-coupling terms.
+
+    *_pos_args :
+        | Temporary shim to update the signature from
+        | ``(..., a_ops, e_ops, c_ops, args, sec_cutoff, options)``
+        | to
+        | ``(..., a_ops, sec_cutoff, *, e_ops, c_ops, args, options)``
+        | making ``e_ops``, ``c_ops``, ``args`` and ``options`` keyword only
+          parameter from qutip 5.3.
```

Comment:
```suggestion
```

I don't know the word "shim", and I don't think these temporary parameters are documented in the other solvers, so we could consider removing this

Reply:
I added it after a comment from Simon. This change is more complex than others so it would help to have a reminder of what the signature previously was somewhere.

Reply:
I see, okay